.github/workflows/git-artifacts.yml

```diff
@@ -418,6 +418,13 @@ jobs:
         with:
           name: arm64-artifacts
           path: ${{github.workspace}}/arm64
+      # Workaround for Git Credential Manager Core on ARM64: https://github.com/git-for-windows/git/issues/3015
+      - name: Create git-credential-manager-core wrapper for ARM64
+        if: matrix.arch.arm64 == true
+        shell: bash
+        run: |
+          printf '%s\n' '#!/bin/sh' 'exec /mingw32/libexec/git-core/git-credential-manager-core.exe "$@"' > "$PWD"/arm64/bin/git-credential-manager-core
+          chmod +x "$PWD"/arm64/bin/git-credential-manager-core
```

Comment:
Is that `"$PWD"/` even needed? I guess it would work without, and simpler is better ;-)

Reply:
You're right. Fixed 👍🏼